File: sdk/src/main/java/com/cxense/cxensesdk/model/UserExternalDataResponse.java
package com.cxense.cxensesdk.model;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

import java.util.Collections;
import java.util.List;

/**
 * Response for data associated with the user(s) from server.
 *
 * @author Dmitriy Konopelkin (dmitry.konopelkin@cxense.com) on (2017-06-05).
 */
public final class UserExternalDataResponse {
    @NonNull
    @SerializedName("data")
    public List<UserExternalData> items = Collections.emptyList();
}

File: sdk/src/main/java/com/cxense/cxensesdk/model/UserExternalDataResponse.kt
package com.cxense.cxensesdk.model

import com.google.gson.annotations.SerializedName

/**
 * Response for data associated with the user(s) from server.
 *
 */
class UserExternalDataResponse(
    @SerializedName("data") val items: List<UserExternalData> = emptyList()
)
